### ISSUE TITLE
`command <someCommand>` missing in several places throughout the shell code

### ISSUE THREAD

**Issue by calestyo** [junegunn/fzf#3458]
- [X] I have read through the manual page (`man fzf`)
- [X] I have the latest version of fzf
- [X] I have searched through the existing issues

## Info

- OS
    - [X] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [X] bash
    - [ ] zsh
    - [ ] fish

## Problem / Steps to reproduce

Hey.

AFAIU, you use `command <someCommand>` in the sourced shell files to prevent undesired alias subsitution, which would likely break everything - just as e.g. bash-completion does as well and which of course is a good thing.

Looking at the shell code (`completion.bash` and `key-bindings.bash` at least), there are however numerous places where this isn't done, e.g.:
https://github.com/junegunn/fzf/blob/a3ff49aaf18eceefe4bf8b1df5bf275b3d982493/shell/completion.bash#L17

https://github.com/junegunn/fzf/blob/a3ff49aaf18eceefe4bf8b1df5bf275b3d982493/shell/completion.bash#L51

https://github.com/junegunn/fzf/blob/a3ff49aaf18eceefe4bf8b1df5bf275b3d982493/shell/completion.bash#L175

https://github.com/junegunn/fzf/blob/a3ff49aaf18eceefe4bf8b1df5bf275b3d982493/shell/completion.bash#L284-L287

I could try to write a patch that adds `command`, but:
- only after my other PRs have been either merged or rejected (otherwise it gets to messy)
- we need to decide on what we protect via `command`

I think for cases like `awk` or `tr` it makes sense to use `command`. I also have seen cases, where people alias `echo`/`printf`. Not sure about things like `read`, `break` or `eval`. I'd tend to perhaps use `command` for `read` but not for any of the special built-ins.

Also, function names like in:
https://github.com/junegunn/fzf/blob/a3ff49aaf18eceefe4bf8b1df5bf275b3d982493/shell/completion.bash#L185
may be aliased, too.<br>
We could in principle avoid alias substitution in such cases by quoting the function name, not sure whether this should be done though.

Cheers,
Chris.

**Comment by junegunn** [comment]
> * we need to decide on what we protect via `command`

That's the hard part. An easy solution is to put `command` only when someone actually complains and raises an issue. That's how we've been handling the issue so far.

> I also have seen cases, where people alias `echo`/`printf`.

Wow.

**Comment by calestyo** [comment]
> That's the hard part.

What if we use command on any program/utility that is not a special built-in command? And, to increase acceptance ;-), also exclude `echo`/`printf` for now.

**Comment by calestyo** [comment]
Well sadly it probably makes sense now to close all my open issues as wontfix, but this one was actually fixed, thus closing.
